test/e2e/unpack_test.go

```diff
@@ -169,11 +173,61 @@ var _ = Describe("Catalog Unpacking", func() {
 			err = c.Get(ctx, client.ObjectKeyFromObject(expectedBndl), bndl)
 			Expect(err).ToNot(HaveOccurred())
 			Expect(expectedBndl).To(komega.EqualObject(bndl, komega.IgnoreAutogeneratedMetadata))
-		})
 
-		AfterEach(func() {
-			err := c.Delete(ctx, catalog)
+			By("Making sure the catalog content is available via the http server")
+			// (TODO): Get the URL from the CR once https://github.com/operator-framework/catalogd/issues/119 is done
+			catalogURL := fmt.Sprintf("%s.%s.svc/catalogs/%s/all.json", "catalogd-catalogserver", defaultSystemNamespace, catalogName)
+			job = batchv1.Job{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-svr-job",
+					Namespace: defaultSystemNamespace,
+				},
+				Spec: batchv1.JobSpec{
+					Template: corev1.PodTemplateSpec{
+						Spec: corev1.PodSpec{
+							Containers: []corev1.Container{
+								{
+									Name:    "test-svr",
+									Image:   "curlimages/curl",
+									Command: []string{"sh", "-c", "curl --silent --show-error --location --insecure - " + catalogURL},
```

Comment:
Oh, I think we deleted a little too much out of this line. And we also don't need `--insecure` anymore.
```suggestion
									Command: []string{"sh", "-c", "curl --silent --show-error --location -o - " + catalogURL},
```

Reply:
Done.